File: 1614. Maximum Nesting Depth of the Parentheses.go
func maxDepth(s string) int {
    count, answer := 0, 0
    for _, c := range(s) {
        if c == '(' {
            count += 1
        } else if c == ')' {
            count -= 1
        }
        if answer < count {
            answer = count
        }
    }
    return answer
}

File: python/1614. Maximum Nesting Depth of the Parentheses.py
class Solution:
    def maxDepth(self, s: str) -> int:
        count, answer = 0, 0
        for i in range(len(s)):
            if s[i] == '(':
                count += 1
            elif s[i] == ')':
                count -= 1
            answer = max(answer, count)
        return answer

